dScripts/RockHydrantSmashable.cpp

```diff
@@ -1,24 +1,23 @@
 #include "RockHydrantSmashable.h"
 #include "EntityManager.h"
-#include "SimplePhysicsComponent.h"
-#include "Entity.h"
-#include "GameMessages.h"
-#include "Game.h"
-#include "dLogger.h"
+#include "GeneralUtils.h"
 
-void RockHydrantSmashable::OnDie(Entity* self, Entity* killer) {
-	SimplePhysicsComponent* physics = self->GetComponent<SimplePhysicsComponent>();
-	NiPoint3 pos = physics->GetPosition();
+void RockHydrantSmashable::OnDie(Entity* self, Entity* killer)
+{
+	const auto hydrantName = self->GetVar<std::u16string>(u"hydrant");
 
-	EntityInfo info;
+	LDFBaseData* data = new LDFData<std::string>(u"hydrant", GeneralUtils::UTF16ToWTF8(hydrantName));
+
+	EntityInfo info {};
 	info.lot = 12293;
```

Comment:
Would be a bit better to give this number a name so it's not just random.  Can define it in the header for this file as a private